etcdutl/snapshot/v3_snapshot.go

```diff
@@ -138,31 +139,42 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 		if v != nil {
 			ds.Version = v.String()
 		}
+		var kv mvccpb.KeyValue
 		c := tx.Cursor()
 		for next, _ := c.First(); next != nil; next, _ = c.Next() {
 			b := tx.Bucket(next)
-			if b == nil {
-				return fmt.Errorf("cannot get hash of bucket %s", string(next))
-			}
-			if _, err = h.Write(next); err != nil {
-				return fmt.Errorf("cannot write bucket %s : %v", string(next), err)
+			_, err = h.Write(next)
+			if err != nil {
+				return fmt.Errorf("cannot hash bucket: name=%q err=%w", string(next), err)
 			}
+
 			iskeyb := (string(next) == "key")
```

Comment:
Would be nice to avoid const and get name from bucket struct itself.

Reply:
Chaged it as `iskeyb := (bytes.Equal(next, schema.Key.Name()))`.